actions/class.ItemImport.php

```diff
@@ -67,4 +66,30 @@ protected function getAvailableImportHandlers()
 
         return $returnValue;
     }
+
+    private function replaceAvailableImportHandlers(): array
+    {
+        $returnValue = parent::getAvailableImportHandlers();
+
+        foreach (array_keys($returnValue) as $key) {
+            if ($returnValue[$key] instanceof \tao_models_classes_import_CsvImporter) {
+                //Check feature flag for tabular import is enabled or not
+                if (!$this->getFeatureFlagChecker()->isEnabled(self::FEATURE_FLAG_TABULAR_IMPORT)) {
+                    // Remove CSV object if tabular import is disabled
+                    unset($returnValue[$key]);
+                } else {
+                    $importer = new CsvItemImporter();
+                    $importer->setServiceLocator($this->getServiceLocator());
+                    $returnValue[$key] = $importer;
+                }
+            }
```

Comment:
Good job! 

Just some clean code recommendations. 

- You can always avoid ELSE keyword for better readability. Code becomes smaller: https://williamdurand.fr/2013/06/03/object-calisthenics/#2-dont-use-the-else-keyword
- You can avoid comments, cause your code is self-explanatory.

```suggestion
            if ($returnValue[$key] instanceof \tao_models_classes_import_CsvImporter) {
                if ($this->getFeatureFlagChecker()->isEnabled(self::FEATURE_FLAG_TABULAR_IMPORT)) {
                    $importer = new CsvItemImporter();
                    $importer->setServiceLocator($this->getServiceLocator());
                    $returnValue[$key] = $importer;
                    
                    continue;
                }
                
                unset($returnValue[$key]);
            }
```